notebooks/LST1_observation_simulator.ipynb

```diff
@@ -0,0 +1,965 @@
+{
```

Comment:
<p>This way of looking for the GAMMAPY_DATA folder in case it is not previously defined is not working for me. I'd say it would work only if you have downloaded the dataset somewhere even if not export the path.</p><p> </p><p>I'd say they way to go is that the user set this up in advance (put some instruction to do so). If running in the container we could simply export the path to a common GAMMAPY_DATASET path.</p><p> </p><p>If running locally I see no way out to downloading and setting GAMMAPY_DATA. Or maybe simply download the <code>ebl_dominguez11.fits.gz</code> and read it, instead of using the read_builtin method.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/1313/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Thanks, I will do this. Since we anyway copy the ebl file, I see no need to check if GAMMAPY_DATA is set or not.